src/methods.jl

```diff
@@ -97,20 +122,35 @@ end
 # The below does not apply to Nonstandard, as it has its own constructor
 for D in (:Forward, :Backward, :Central)
     lcname = lowercase(String(D))
-    gridf = Symbol(lcname, "_grid")
-    fdmf = Symbol(lcname, "_fdm")
+    gridf = Symbol(lcname, :_grid)
+    fdmf = Symbol(lcname, :_fdm)
+    precomputes = Symbol(lcname, :_precomputed)
 
+    num_precomputed = 8
     @eval begin
         # Compatibility layer over the "old" API
         function $fdmf(p::Integer, q::Integer; adapt=1, kwargs...)
             _dep_kwarg(kwargs)
             return $D(p, q; adapt=adapt, kwargs...)
         end
 
+        # precompute a bunch of grids and coefs
+        const $precomputes = ntuple($num_precomputed) do p
```

Comment:
Should this be defined in a constant?
Or should it be declared outside the eval, and then inlined in as a local literal
and thus should constant fold